File: atcoder/abc162B_fizzbuzz_sum.py
# uninhm
# https://atcoder.jp/contests/abc162/tasks/abc162_b
# simulation
ans = 0
for i in range(int(input())+1):
    if not (i%3 == 0 or i%5 == 0):
        ans += i
print(ans)

File: atcoder/abc162B_fizzbuzz_sum.ml
(* uninhm
 * https://atcoder.jp/contests/abc162/tasks/abc162_b
 * implementation
 * *)

Printf.printf "%d\n" @@
Scanf.scanf "%d\n" @@ fun n ->
    let ans = ref 0 in
    for i = 1 to n do
        if (i mod 3 != 0) && (i mod 5 != 0) then
            ans := !ans + i;
    done;
    !ans
